### PR TITLE
Add `read-amp-bytes-per-bit` option

### PR DESCRIPTION
**Pull request by huachaohuang**

### CODE REVIEW

**Comment by siddontang** [comment]
LGTM

File: src/config.rs [hunk 1]
```diff
@@ -97,6 +98,7 @@ macro_rules! build_cf_opt {
                                              $opt.block_based_bloom_filter);
             block_base_opts.set_whole_key_filtering($opt.whole_key_filtering);
         }
+        block_base_opts.set_read_amp_bytes_per_bit($opt.read_amp_bytes_per_bit);
```
Comment (siddontang) [inline src/config.rs]: please add an example in the etc config too.

**Comment by huachaohuang** [comment]
PTAL @zhangjinpeng1987

File: etc/config-template.toml [hunk 2]
```diff
@@ -301,6 +301,15 @@
...
 # pin-l0-filter-and-index-blocks = true
 
+# Enable read amplication statistics.
+# value  =>  memory usage (percentage of loaded blocks memory)
+# 1      =>  12.50 %
+# 2      =>  06.25 %
+# 4      =>  03.12 %
+# 8      =>  01.56 %
+# 16     =>  00.78 %
+# read-amp-bytes-per-bit = 16
```
Comment (zhangjinpeng1987) [inline etc/config-template.toml]: By default we use `0`, please keep consistent between code and config.toml.

**Pull request merged**